schema/common/defaults/configuration/opensearch.yml

```diff
@@ -0,0 +1,32 @@
+kind: configuration/opensearch
+title: OpenSearch Config
+name: default
+specification:
+  cluster_name: EpiphanyOpenSearch
+  opensearch_os_user: opensearch
+  opensearch_os_group: opensearch
+  admin_password: PASSWORD_TO_CHANGE
+  kibanaserver_password: PASSWORD_TO_CHANGE
+  kibanaserver_user_active: true
+  logstash_password: PASSWORD_TO_CHANGE
+  logstash_user_active: true
+  demo_users_to_remove:
+    - kibanaro
+    - readall
+    - snapshotrestore
+    # - logstash
+    # - kibanaserver
+  paths:
+    opensearch_home: /usr/share/opensearch
+    opensearch_conf_dir: /usr/share/opensearch/config
+    opensearch_log_dir: /var/log/opensearch
+    opensearch_repo: /var/lib/opensearch-snapshots
+    opensearch_data: /var/lib/opensearch
+    opensearch_logs: /var/log/opensearch
+    opensearch_perftop_home: /usr/share/opensearch/perftop
```

Comment:
What would you say about removing opensearch prefixes in these paths?
It is already nested in opensearch specification and paths are only for opensearch here.

Reply:
Yes, it's nested but we are not referencing this way: `opensearch.specification.paths.opensearch_home` but this way:
`specification.paths.opensearch_home`. To keep it readable I would leave it as is.